package.json

```diff
@@ -259,6 +263,11 @@
 					"command": "gerrit.openOriginal",
 					"when": "view == gerrit:changeExplorer && viewItem =~ /filechange/ && viewItem =~ /modified/",
 					"group": "open@3"
+				},
+				{
+					"command": "gerrit.checkoutBranch",
+					"when": "view == gerrit:changeExplorer && viewItem =~ /change/",
```

Comment:
FYI in the latest branch I renamed this from `change` to `gerritchange` on the master branch so you should update this when you rebase